spec/order_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/order.rb'
+require './lib/dish_list.rb'
+
+describe Order do
+
+  let(:order) { Order.new(menu) }
+  let(:menu) { double(:dish_list, :menu => "Pizza - 6.5") }
+  let(:dish_list_class) { double(:dish_list_class, :new => menu) }
+
+  it 'can show all dishes with prices in a nice format' do
+    order
+    expect(order.view_dishes).to eq("Pizza - 6.5")
+  end
+
+  it 'can select a specific dish and display price' do
+    o = Order.new
```

Comment:
You're super close with mocking here! 
Wondering if there is a reason you chose to create a new instance of `Order` here? If you used the `order` variable defined on line 6 you could make use of the mocked `dish_list_class` instead?